### PR TITLE
Improve message html rendering

### PR DESCRIPTION
**Pull request by borisyankov**
* Use only `View` components instead of mixing `View` and `Text`
* Can render list inside a list
* Tweaked sizes and styles for several components

### CODE REVIEW

File: src/html/cascadingStylesText.js [hunk 1]
```diff
@@ -37,11 +27,11 @@ export default StyleSheet.create({
   },
   code: {
     fontFamily: 'Menlo',
-    fontSize: 13,
+    // fontSize: 13,
```
Comment (neerajwahi) [inline src/html/cascadingStylesText.js]: Please remove the commented out code

File: src/html/HtmlNodeText.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import entities from 'entities';
 import { Text } from 'react-native';
 
-import styles from '../../styles';
+// import styles from '../../../styles';
```
Comment (neerajwahi) [inline src/html/HtmlNodeText.js]: Please remove comment

File: src/html/HtmlNodeTag.js [hunk 3]
```diff
@@ -55,14 +55,6 @@ export default ({ auth, attribs, name, cascadingStyle, childrenNodes, onPress })
 
   const HtmlComponent = specialTags[name] || HtmlTagSpan;
 
-  // User mentions should not have any breaking spaces
```
Comment (neerajwahi) [inline src/html/HtmlNodeTag.js]: Why remove this?

Comment (borisyankov) [inline src/html/HtmlNodeTag.js]: The current implementation does not need it. It works like this by default.

**Comment by borisyankov** [comment]
@neerajwahi I did fix the mentioned issues. Good to merge?

**Pull request merged**